package.json

```diff
@@ -1,9 +1,10 @@
 {
   "scripts": {
-    "prepare": "husky install",
-    "load-directus": "docker exec -it clic-directus npx directus schema apply /share/snapshot.yaml&&docker restart clic-directus",
-    "save-directus": "sh save-directus.sh",
-    "populate-directus": "docker exec -i clic-postgres psql --user directus_user directus_data < ./directus/dump.sql"
+    "prepare": "husky install && git submodule update",
```

Comment:
Doesn't this miss the `--init --reccursive` (or just `--init` in our case) flags for the `git submodule update command` ?